Rover/mode.cpp

```diff
@@ -414,6 +414,14 @@ float Mode::calc_speed_nudge(float target_speed, bool reversed)
 // this function updates _distance_to_destination
 void Mode::navigate_to_waypoint()
 {
+    if ( g2.sailboat.in_irons()){
+       // set rudder output to turn the boat in required direction
+       // rudder_percent ( 0.0f to 1.0f) sets the rudder angle used to turn the boat
+       // as it is blown backwards. Too much rudder angle can stall the rudders
+       // and slow the process of getting out of irons
+       g2.motors.set_steering(g2.sailboat.get_in_irons_rudder() * 4500.0f,false);
```

Comment:
```suggestion
       g2.motors.set_steering(g2.sailboat.get_in_irons_rudder() * 4500.0, false);
```

Reply:
If you insist but I always prefer to be explicit about float v double. set_steering() takes a float, so give it a float not a double. Prevents unnecessary conversions and hidden narrowing conversions
I don't know the official ardupilot policy about that though Ardupilot style guide is AWOL https://dev.ardupilot.org/docs/style-guide.html  ( gives me a "security risk" warning)

Reply:
I will leave these  for now, unless you object strongly. Ultimately whichever you prefer but If you want all the  explicit float literals removed throughout the codebase, maybe a bigger PR